openverse_catalog/dags/maintenance/check_silenced_dags/check_silenced_dags.py

```diff
@@ -48,10 +53,10 @@ def check_configuration(github_pat: str, airflow_variable: str):
 
     message = (
         "The following DAGs have Slack messages silenced, but the associated issue is"
-        f" closed. Please remove them from the `{airflow_variable}` Airflow variable"
-        " or assign a new issue."
+        " closed. Please remove them from the silenced_slack_notifications Airflow"
+        " variable or assign a new issue."
     )
-    for (dag, issue) in dags_to_reenable:
-        message += f"\n  - <{issue}|{dag}>"
+    for (dag, issue, predicate) in dags_to_reenable:
+        message += f"\n  - <{issue}|{dag}: {predicate}>"
```

Comment:
I'm not sure this will work, but it might be a good idea to quote this predicate for easier reading:
```suggestion
        message += f"\n  - <{issue}|{dag}: '{predicate}'>"
```